### ISSUE TITLE
clean up test.rviz

### ISSUE THREAD

**Issue by jpreiss** [USC-ACTLab/crazyswarm#340]
our rviz configuration file https://github.com/USC-ACTLab/crazyswarm/blob/master/ros_ws/src/crazyswarm/launch/test.rviz appears more complicated than necessary. It has some configurations for specific IDs. Also, a name other than "test" would be more descriptive.

**Comment by whoenig** [comment]
Agreed about re-naming. Other than that, this file is generated by rviz, and not handwritten, so I am not sure if we really should spend effort in making it look clean.
